src/plan_bot.py

```diff
@@ -314,3 +345,30 @@ def create_db_record(
     }
 
     return entry
+
+
+def get_trigger_line(text, trigger_word="!warrenplanbot"):
+    """
+    Get the last line that !WarrenPlanBot occurs on,
+    only returning the part of that line which occurs _after_ !WarrenPlamBot
+    """
+    matches = re.findall(fr"{trigger_word}[^-\w]+(.*)$", text, re.IGNORECASE | re.MULTILINE)
+
+    return matches[-1] if matches else ""
+
+
+def process_flags(text):
+    """
+    Identifies flags in the text. Removes the flags from the text and
+    returns the tuple (remaining_text, options).
+    """
+    options = {
+        "parent": False
+    }
+
+    match = re.match(r"^(?:-p|--parent)[^-\w]+(.*)$", text, re.IGNORECASE | re.MULTILINE)
```

Comment:
I think based on earlier discussion, let's avoid the `-p` form. Too likely someone will copy-pasta that without knowing what they are doing imo. If the feature gets popular, we can add that in later as a shortcut